src/extension.ts

```diff
@@ -5,20 +5,26 @@ import { AustinController } from './controller';
 import { AustinStats } from './model';
 import { TopDataProvider } from './providers/top';
 import { CallStackDataProvider } from './providers/callstack';
+import { AustinProfileTaskProvider } from './providers/task';
+import { AustinRuntimeSettings } from './settings';
+import { AustinMode } from './types';
 
 
 export function activate(context: vscode.ExtensionContext) {
-
 	vscode.workspace.onDidChangeTextDocument((changeEvent) => {
 		clearDecorations();
 	});
 
+	const output :vscode.OutputChannel = vscode.window.createOutputChannel("Austin");
+	const workspaceRoot = vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders[0].uri.fsPath : undefined;
+	
 	const stats = new AustinStats();
-	const controller = new AustinController(stats);
+	const controller = new AustinController(stats, output);
 
 	const flameGraphViewProvider = new FlameGraphViewProvider(context.extensionUri);
 	const topProvider = new TopDataProvider();
 	const callStackProvider = new CallStackDataProvider();
+	const austinProfileProvider = new AustinProfileTaskProvider(output, stats, workspaceRoot);
```

Comment:
It looks like we should be able to get the value for `workspaceRoot` directly from the `AustinProfileTaskProvider` constructor instead of passing it as an argument?